services/ucs/src/ucs/core/models.py

```diff
@@ -15,85 +15,51 @@
-        - REJECTED (the upload was rejected by a downstream service)
-    """
-
-    PENDING = "pending"
-    CANCELLED = "cancelled"
-    UPLOADED = "uploaded"
-    FAILED = "failed"
-    ACCEPTED = "accepted"
-    REJECTED = "rejected"
-
-
-class UploadAttempt(BaseModel):
-    """A model containing details on an upload attempt for a specific File."""
-
-    upload_id: str
-    file_id: str = Field(
-        default=..., description="The ID of the file corresponding to this upload."
-    )
-    object_id: str = Field(
-        default=...,
-        description="The bucket-specific ID used within the S3 object storage.",
-    )
-    status: UploadStatus
-    part_size: int = Field(
-        default=..., description="Part size to be used for upload. Specified in bytes."
-    )
-    creation_date: UTCDatetime = Field(
-        default=..., description="Datetime when the upload attempt was created."
-    )
-    completion_date: UTCDatetime | None = Field(
-        default=None,
-        description=(
-            "Datetime when the upload attempt was declared as completed by the client."
-            + " `None` if the upload is ongoing."
-        ),
-    )
-    submitter_public_key: str = Field(
-        default=...,
-        description="The public key used by the submittter to encrypt the file.",
-    )
-    model_config = ConfigDict(from_attributes=True, title="Multi-Part Upload Details")
-    storage_alias: str = Field(
-        default=...,
-        description="Alias for the object storage location where the given object is stored.",
-    )
-
-
-class FileMetadataUpsert(BaseModel):
-    """A model for creating new or updating existing file metadata entries."""
-
-    file_id: str
-    file_name: str
-    decrypted_sha256: str
-    decrypted_size: int
-    model_config = ConfigDict(from_attributes=True, title="File Metadata Creation")
-
-
-class FileMetadata(FileMetadataUpsert):
-    """A model containing the full metadata on a file."""
-
-    latest_upload_id: str | None = Field(
-        default=None,
-        description=(
-            "ID of the latest upload (attempt). `Null/None`"
-            + " if no update has been initiated, yet."
-        ),
-    )
-    model_config = ConfigDict(from_attributes=True, title="File Metadata")
+from pydantic import UUID4, BaseModel
+
+
+class FileUploadBox(BaseModel):
+    """A class representing a box that bundles files belonging to the same upload"""
+
+    id: UUID4  # unique identifier for the instance
+    locked: bool = False  # Whether or not changes to the files in the box are allowed
+    file_count: int = 0  # The number of files in the box
+    size: int = 0  # The total size of all files in the box
+    storage_alias: str
+
+
+FileUploadState = Literal["init", "inbox", "archived"]
+
+
+class FileUpload(BaseModel):
+    """A File Upload"""
+
+    id: UUID4  # unique identifier for the instance
+    completed: bool = False  # whether or not the file upload has finished
```

Comment:
Why do you still keep the `completed` field when `state` carries the same information now?